### PR TITLE
[MIG] purchase_third_validation: migrated this module from 8.0 to 11.… task#6776

### PR DESCRIPTION
**Pull request by keylor2906**
…0, this module enables a third validation on purchase orders in case it is needed a new limit amount.

### CODE REVIEW

**Comment by keylor2906** [comment]
@josemoralesp could you review this, please?

**Comment by sabrinagnu** [comment]
@josemoralesp where is the dummy?, this also need my functional Vo Bo before merge.

File: purchase_third_validation/models/purchase.py [hunk 1]
```diff
@@ -1,47 +1,23 @@
...
+    def button_approve(self, force=False):
+        group = 'purchase_third_validation.general_purchase_manager'
+        limit = self.env.user.company_id.currency_id.compute(
+            self.company_id.po_third_validation_amount, self.currency_id)
+        if self.company_id.po_double_validation in ['one_Step', 'two_step']\
+                or (self.company_id.po_double_validation == 'three_step'
+                    and self.amount_total < limit)\
+                or self.user_has_groups(group):
+            return super(PurchaseOrder, self).button_approve(force)
+        else:
```
Comment (luistorresm) [inline purchase_third_validation/models/purchase.py]: The `if` has a return, you do not define the `else` sentence?

**Comment by keylor2906** [comment]
@dsabrinarg here is the dummy pr on lodi https://git.vauxoo.com/vauxoo/lodi/merge_requests/22

**Comment by sabrinagnu** [comment]
@josemoralesp Functional tests 👍

**Pull request merged**